template/cordova/lib/deployment.js

```diff
@@ -194,13 +194,28 @@ AppDeployCmdTool.prototype.enumerateDevices = function() {
     });
 };
 
+// Note: To account for CB-9482, we pass an extra parameter when retrying the call.  Be forwarned to check for that
+// if additional parameters are added in the future.
 AppDeployCmdTool.prototype.installAppPackage = function(pathToAppxPackage, targetDevice, shouldLaunch, shouldUpdate, pin) {
     var command = shouldUpdate ? '/update' : '/install';
     if (shouldLaunch) {
         command += 'launch';
     }
 
-    return run(this.path, [command, pathToAppxPackage, '/targetdevice:' + targetDevice.__shorthand]);
+    var that = this;
+    var result = run(this.path, [command, pathToAppxPackage, '/targetdevice:' + targetDevice.__shorthand]);
+    if (targetDevice.type === 'emulator' && arguments.length < 6) {
```

Comment:
It's called out in the comment immediately preceding the function definition.  There are 5 formal arguments specified to the function, so when the function is called normally, arguments.length = 5.  In that case, we compose an error handler onto result; the composed error handler is only called if AppDeployCmd fails.  That handler checks the code to see if it's `E_INVALIDARG`, and if so, re-calls the function, this time passing an additional parameter.